src/Http/MiddlewareQueue.php

```diff
@@ -55,37 +57,15 @@ public function __construct(array $middleware = [])
         $this->queue = $middleware;
     }
 
-    /**
-     * Get the middleware at the provided index.
-     *
-     * @param int $index The index to fetch.
-     * @return \Psr\Http\Server\MiddlewareInterface|null Either the middleware or null
-     *   if the index is undefined.
-     */
-    public function get(int $index): ?MiddlewareInterface
```

Comment:
Do you think someone will be using this? We haven't deprecated this method in 3.7.

Reply:
I don't see any reason why someone would need to get a middleware from queue themselves.

Reply:
We can mark it deprecated in 4.0 and keep it until 5.0 but I really don't see the need.

Reply:
Ok. Let's remove it and see what happens.  :smile;

Reply:
Cool, this should be good to merge then :slightly_smiling_face:.